specification/columns/servicesubcategory.md

```diff
@@ -2,6 +2,18 @@
 
 The Service Subcategory is a secondary classification of the [Service Category](#servicecategory) for a [*service*](#glossary:service) based on its core function. The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing spend and usage for specific workload types across providers and tracking the migration of workloads across fundamentally different architectures.  
 
+---
+The ServiceSubcategory column adheres to the following requirements:
+
+* ServiceSubcategory is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
+* If present, the column adheres to the following additional requirements:
+  * ServiceSubcategory MUST be of type String.
+  * ServiceSubcategory MUST NOT be null.
+  * ServiceSubcategory MUST be one of the allowed values.
+  * ServiceSubcategory MUST have one and only one parent ServiceCategory as specified in the allowed values below.
+  * Though a given *service* can have multiple purposes, each *service* SHOULD have one and only one ServiceSubcategory that best aligns with its primary purpose
```

Comment:
This is a confusing requirement. We have allowed values, so it should be a MUST. If this is context on the allowed values requirement, I would rather see it nested under that and remove the "SHOULD".